controllers/user.js

```diff
@@ -72,6 +72,19 @@ async function updateCommunicationOptions(req, res) {
 }
 
 
+async function resolveBreach(req, res) {
+  const sessionUser = req.user;
+  // TODO: verify that req.body.emailAddressId belongs to sessionUser
+  const updatedSubscriber = await DB.setResolvedBreach({
+    subscriber: sessionUser,
+    emailAddresses: req.body.emailAddressId,
+    recencyIndex: req.body.recencyIndex,
```

Comment:
Q: Do we need to verify that the `req.body` object includes the `emailAddressId` and `recencyIndex`?